Services/LeverPostingService.cs

```diff
@@ -101,9 +102,11 @@ private async Task<IList<ContentItem>> CreateUpdateAsync(IList<Posting> postings
             var postingContentItems = await GetAllAsync();
 
             // Remove old postings
+            _logger.LogInformation($"Lever: Remove content items from API");
```

Comment:
This should be moved to within the `RemoveAsync` method and contain a count of the number of postings that are being removed.